extensions/wikia/Recirculation/js/trackers/liftigniter.js

```diff
@@ -17,3 +17,4 @@ $p("init", "l9ehhrb6mtv75bp2", {
     }
 });
 $p("send", "pageview");
+$p("setRequestFields", ["rank", "thumbnail", "title", "url", "presented_by", "author"]);
```

Comment:
Ideally each request to `model.json` would have its own values of `setRequestFields`, since now we're going to want to fetch different data for wiki recommendations (wiki name, impressions, etc) than we do for news+stories. As it is now I think we'll have to have a big call `setRequestFields`, something like `$p('setRequestFields', ['rank', 'thumbnail', 'title', 'url', 'presented_by', 'author', 'wikiImpressions', 'wikiDomain', ...])`